backend/core/src/listings/listings.service.ts

```diff
@@ -137,6 +137,11 @@ export class ListingsService {
         delete unit.id
       }
     })
+    listingDto.unitsSummary.forEach((summary) => {
+      if (summary.id.length === 0 || summary.id === "undefined") {
+        delete summary.id
+      }
+    })
```

Comment:
This will remove the unitsSummary id entirely from the listing if that one is empty, I take it?

Reply:
That's correct